Cargo.toml

```
@@ -69,7 +69,7 @@ jubjub-schnorr = { version = "0.6", default-features = false }
 kadcast = "0.7"
 phoenix-circuits = { version = "0.6", default-features = false }
 phoenix-core = { version = "0.34.0", default-features = false }
-piecrust = "0.27.2"
+piecrust = "0.28.0-rc.0"
```

```suggestion
piecrust = "0.28.0-rc.6"
```

Otherwise if can fail

---

done